projects/serverless-scraping/terraform/main.tf

```
@@ -0,0 +1,165 @@
+terraform {
+  required_providers {
+    scaleway = {
+      source = "scaleway/scaleway"
+    }
+  }
+  required_version = ">= 0.13"
+}
+
+provider "scaleway" {
+}
+
+resource "scaleway_account_project" "mnq_tutorial" {
+  name = "mnq-tutorial"
+}
+
+# ============= SQS ===============
+resource "scaleway_mnq_sqs" "main" {
+  project_id = scaleway_account_project.mnq_tutorial.id
+}
+
+resource "scaleway_mnq_sqs_credentials" "producer_creds" {
+  project_id = scaleway_mnq_sqs.main.project_id
+  name = "sqs-credentials-producer"
+
+  permissions {
+    can_manage  = true
+    can_receive = false
+    can_publish = true
+  }
+}
+
+resource "scaleway_mnq_sqs_credentials" "consumer_creds" {
+  project_id = scaleway_mnq_sqs.main.project_id
+  name = "sqs-credentials-consumer"
+
+  permissions {
+    can_manage  = false
+    can_receive = true
+    can_publish = false
+  }
+}
+
+resource "scaleway_mnq_sqs_queue" "main" {
+  # possible to pass scaleway_mnq_sqs resource directly? To see with devtools
+  project_id = scaleway_account_project.mnq_tutorial.id
+  name       = "hn-queue"
+  access_key = scaleway_mnq_sqs_credentials.producer_creds.access_key
+  secret_key = scaleway_mnq_sqs_credentials.producer_creds.secret_key
+}
+
+# ============= RDB ===============
+
+resource "random_password" "dev_mnq_pg_exporter_password" {
+  length           = 16
+  special          = true
+  min_numeric      = 1
+  min_upper        = 1
+  min_lower        = 1
+  min_special      = 1
+  override_special = "_-"
+}
+
+output db_password {
+  value = random_password.dev_mnq_pg_exporter_password.result
+  sensitive = true
+}
+
+resource "scaleway_rdb_instance" "main" {
+  name = "test-rdb"
+  project_id   = scaleway_account_project.mnq_tutorial.id
+  node_type = "db-dev-s"
+  engine = "PostgreSQL-15"
+  is_ha_cluster = false
+  disable_backup = true 
+  user_name = "mnq_initial_user"
+  password = random_password.dev_mnq_pg_exporter_password.result
+}
+
+resource "scaleway_rdb_database" "hn-database" {
```

```suggestion
resource "scaleway_rdb_database" "main" {
```

Usually if there's just one instance of a resource type I call it `main`. It makes things slightly easier to read